ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -6034,10 +6034,10 @@ static int compareTrackedSourceByName(CInterface * const * _left, CInterface * c
     return stricmp(leftName, rightName);
 }
 
-IPropertyTree * HqlCppTranslator::gatherFieldUsage(const char * varient, const IPropertyTree * exclude)
+IPropertyTree * HqlCppTranslator::gatherFieldUsage(const char * variant, const IPropertyTree * exclude)
 {
     Owned<IPropertyTree> sources = createPTree("usedsources");
-    sources->setProp("@varient", varient);
+    sources->setProp("@varient", variant);
```

Comment:
Not in the same PR you shouldn't. You can open a separate Jira for that if you want...

Perhaps we should set both variants (varients) of the tag for back-compatibility...